utils/src/main/java/com/salesforce/op/utils/stats/StreamingHistogram.java

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
I think you need to add `// scalastyle:off header.matches` line here.

Reply:
and also include it to the exclusion list - https://github.com/salesforce/TransmogrifAI/blob/master/build.gradle#L205

Reply:
This class won't go through `scalastyle` since in `src/main/java` so I don't think that's needed.